app/mailers/user_mailer.rb

```diff
@@ -264,7 +264,7 @@ def coauthor_notification(user_id, creation_id, creation_class_name)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: @user.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Co-Author Notification"
+        subject: "#{t 'user_mailer.coauthor_notification.subject', app_name: ArchiveConfig.APP_SHORT_NAME}"
```

Comment:
This should probably be just `t('user_mailer.coauthor_notification.subject', app_name: ArchiveConfig.APP_SHORT_NAME)` like the others